src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -61,6 +65,13 @@ private static ArgumentParser getArgumentParser() {
                 .setDefault(Optional.empty())
                 .help("The date to stop filtering.");
 
+        parser.addArgument("-formats")
+                .nargs("*")
+                .metavar("FORMAT")
+                .setDefault(DEFAULT_FORMATS)
+                .help("The file formats to process. "
```

Comment:
Are there are restrictions for file extension format? Like must it be only alphanumeric or any alphabetic? Can we add this specification here?